### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by achingbrain**
Updates to new buffer version among other things to reduce the amount
of bloat in browser bundles.

### CODE REVIEW

**Review by rvagg — APPROVED** [review]

**Comment by rvagg** [comment]
CI failures for Windows are a weird timeout - the tests are passing just fine but exit doesn't seem to be registered.

The browser failures look identical to what I hit @ https://github.com/ipfs/js-ipfs-unixfs/pull/116#issuecomment-825646616, maybe try switching from `bionic` to `focal` and 🤞.

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,31 @@
 language: node_js
-cache: npm
-stages:
-  - check
-  - test
-  - cov
+cache: false
+dist: bionic
```
Comment (achingbrain) [inline .travis.yml:3]:
```suggestion
dist: focal
```

**Comment by rvagg** [comment]
nope that didn't work .. I was having trouble reproducing the case locally when trying to narrow down something to report to `esbuild` but couldn't do it, there's something really quirky with the conditions in which it removes `typeof require === "function"` in dirty-chai, but I can't find it in their code (yet).

**Comment by achingbrain** [comment]
Hmm, it's definitely esbuild related - something that has changed between `0.11.5` and `0.11.15`.

aegir is [pinned to 0.11.5](https://github.com/ipfs/aegir/blob/master/package.json#L85) but `playwright-test` has [^0.11.5](https://github.com/hugomrdias/playwright-test/blob/master/package.json#L50) which resolves to `0.11.15` right now.

If I run the browser tests on Mac OS X I get the failure, but if I manually dedupe `esbuild` so only `0.11.5` is present in `node_modules` everything passes.

cc @hugomrdias

**Comment by achingbrain** [comment]
`v0.11.10` seems to be the last version that works properly, the error appears with `esbuild@0.11.11`.

**Comment by achingbrain** [comment]
I've opened an issue here: https://github.com/evanw/esbuild/issues/1202

**Pull request merged**